Documentation/Blazorise.Docs/Pages/Blog/2022-09-13_SettingUpABlazorProject/Index.md

```diff
@@ -0,0 +1,117 @@
+
+Currently the SDKs that are compatible with Blazor Applications are
+
+- .NET 7 [Currently available in preview](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-7.0.100-preview.7-windows-arm64-installer)
+- .NET 6 [LTS](https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0.8/6.0.8.md?WT.mc_id=dotnet-35129-website)
+- .NET 5 [Not Supported](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-5.0.408-windows-x86-installer)
+
+### An Integrated Development Environment
+
+A tool that is designed with the necessary tooling needed for the building, testing and deployment of code.
+
+- [Rider by JetBrains](https://www.jetbrains.com/rider/buy/?fromIDE#personal)
+- [Visual Studio for PC/Mac](https://aka.ms/blazor-vs-install) 
+
+For code editors such as Visual Studio Code, running commands in the terminal uses the .NET CLI to produce the same results.  
+
+The .NET command-line interface (CLI) is a cross-platform toolchain for developing, building, running, and publishing .NET applications.
+
+## Types of Blazor Applications
+
+There are two types of Blazor Applications.
+
+Blazor WebAssembly (WASM) is a client-side blazor application that runs in the browser.
+
+> Read More About [Blazor WebAssembly](/blog/what-is-blazorwasm)
+
+Blazor Server is a server-side application that processes client-side request
+
+## Create Up A Blazor Application With IDE
+
+For this example, we will demonstrate how to use the Visual Studio IDE to create a Blazor Application. You can use 
+
+- .NET 5.0 requires Visual Studio 2019 16.8 or later.
+- .NET 6.0 requires Visual Studio 2022 17.0 Preview 4.1 or later.
+
+Open your Visual Studio IDE after installation and create a new project
+
+![Create project](img/blog/2022-09-13/create-project.png)
+
+In the Create New Project Dialog box, search for Blazor. Select Blazor WebAssembly App to create a blazor client-side application or a Blazor Server App to create a server-side blazor application.
+
+![Pick Blazor](img/blog/2022-09-13/pick-blazor.png)
+
+Click the next button to continue to the project configuration page. Here, you can name your Blazor application and select the target framework plus a host of other configurations.
+
+Leave all default selections and click create button. This will scaffold your application into a solution directory.
+
+![Solution explorer](img/blog/2022-09-13/solution-explorer-after-project-create.png)
+
+> In naming your application, always begin the first letter in uppercase. This is more of a best practice than a rule. If your app name contains two or more words, write the first letter of each word in uppercase, as with our example `BlazorApp`.
+
+## Create A Blazor Application With .NET CLI
+
+You can the .NET CLI to convert a command-line application into a headless IDE to create Blazor applications. Examples of a command-line program includes Powershell and CMD on Windows and Terminal on MacOS and Linux.
```

Comment:
The sentence `You can the .NET CLI to convert a command-line application into a headless IDE to create Blazor applications` feels weird.